lib/index.js

```diff
@@ -343,3 +360,14 @@ function _updateSessionAuthenticationData(
   }
   return data;
 }
+
+function _generateRandomSalt({length}) {
+  const bf = crypto.randomBytes(Math.ceil(length));
+  return bf.toString('base64').slice(0, length);
+}
```

Comment:
This differs from the implementation in bedrock-authn-token.
https://github.com/digitalbazaar/bedrock-authn-token/blob/main/lib/pbkdf2.js#L58
https://github.com/digitalbazaar/bedrock-authn-token/blob/main/lib/pbkdf2.js#L166-L177
https://github.com/digitalbazaar/bedrock-authn-token/blob/main/lib/pbkdf2.js#L133-L135

Reply:
Update generate salt method following bedrock-authn-token package.